packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -81,6 +81,12 @@ class ConfigPaymentAgent {
     @observable selected_bank = 0;
     @observable supported_banks = [];
     @observable verification = new ConfigVerification();
+    @observable active_index = 0;
```

Comment:
Isn't it better to have a more descriptive name? `active_index` of what?
What do you think about `active_tab_index`?

Reply:
sure, will change the name